cpp_linter/rest_api/github_api.py

```diff
@@ -522,7 +571,178 @@ def _dismiss_stale_reviews(self, url: str):
                         url=f"{url}/{review['id']}/dismissals",
                         method="PUT",
                         data=json.dumps(
-                            {"message": "outdated suggestion", "event": "DISMISS"}
+                            {"message": "Outdated review", "event": "DISMISS"}
                         ),
                         strict=False,
                     )
+
+    def _get_existing_review_comments(self, no_dismissed: bool = True):
+        """Creates the list existing conversation threads to close.
+
+        :param no_dismissed: `True` to ignore any already dismissed comments.
+        """
+        repo_owner, repo_name = self.repo.split("/")
+        query = """
+        query {
```

Comment:
I feel like these queries could be constants scoped to rest_api/github.py
```py
QUERY_REVIEW_COMMENTS = """ ... """
RESOLVE_REVIEW_COMMENT = """ ... """
DELETE_REVIEW_COMMENT = """ ... """
HIDE_REVIEW_COMMENT = """ ... """
```

Reply:
done